code/renderers/react/src/docs/jsxDecorator.tsx

```diff
@@ -141,7 +142,7 @@ export const renderJsx = (code: React.ReactElement, options: JSXOptions) => {
     return string;
   }).join('\n');
 
-  return result.replace(/function\s+noRefCheck\(\)\s+\{\}/, '() => {}');
+  return result.replace(/function\s+noRefCheck\(\)\s+\{\}/g, '() => {}');
```

Comment:
Before:
<img width="1074" alt="Screen Shot 2022-08-25 at 12 47 42" src="https://user-images.githubusercontent.com/31362988/186577333-fbc6496e-3e37-4040-a973-c267d6724d66.png">

After:
<img width="1074" alt="Screen Shot 2022-08-25 at 12 46 58" src="https://user-images.githubusercontent.com/31362988/186577366-067f454e-78f5-469b-96eb-e4e23e098753.png">

Reply:
It still isn't showing the actual function. Is that the expectedd behavior?